simplivity/resources/policies.py

```diff
@@ -89,6 +92,33 @@ def create(self, name, flags=None, timeout=-1):
         affected_object = self._client.do_post(URL, data, timeout, flags=flags)[0]
         return self.get_by_id(affected_object["object_id"])
 
+    def suspend(self, target_type, target=None, timeout=-1):
+        """Suspends policy-based backups on a specific targeted object
+            Args:
+                target_type: The type of object where policy backups should be suspended
+                             allowed values are host, omnistack_cluster, cluster_group, federation
+                target: Target object/name
```

Comment:
@sijeesh what do you think if we required target to be an object of type host, omnistack_cluster, cluster_group, or of string type called "federation" to cover the odd man out edge case?

This would simplify the suspend call for a better total customer user experience.

We should be able to determine the target_type by using the type of the object passed, and if it was a string passed in we would be able to set the target_type, and address that according.

Reply:
@svt-kmcdaniel, I think it's better to remove target_type and accept only objects of the omnistack_cluster, cluster_group, and host for target argument. We can find the target_type from the target object.

Reply:
Thanks @sijeesh , @svt-kmcdaniel 
Updated the code as suggested